blazor/pager/data-binding.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title: Data Binding in Blazor Pager Component | Syncfusion
+description: Checkout and learn here all about working with data in Syncfusion Blazor Pager component and much more.
+platform: Blazor
+control: Pager
+documentation: ug
+---
+
+# Data Binding in Blazor Pager Component
+
+## Local Data
+
+Local data can be represented in Array of JSON data:
```

Comment:
in blazor we dont provide the data in JSON format. It is like list format only so please change the sentence accordingly. get the content from here data binding topic
https://www.syncfusion.com/blazor-components/blazor-datagrid